platforms/quorum/configuration/molecule/ambassador-certs/converge.yml

```diff
@@ -0,0 +1,144 @@
+        retry_count: 20                
+        external_dns: enabled
+
+      organizations:
+        - organization:
+          name: carrier
+          unit: Carrier
+          external_url_suffix: test.quorum.blockchaincloudpoc.com 
+          cloud_provider: aws 
+          aws:
+            access_key: "aws_access_key" 
+            secret_key: "aws_secret_key" 
+          k8s:
+            context: "cluster_context"
+            config_file: "cluster_config"
+          vault:
+            url: "vault_addr"
+            root_token: "vault_root_token"
+            secret_path: "secret"
+            
+          gitops:
+            git_protocol: "https" 
+            git_url: "https://github.com/<username>/blockchain-automation-framework.git"
+            branch: "develop" 
+            release_dir: "platforms/quorum/releases/dev" 
+            chart_source: "platforms/quorum/charts"  
+            git_repo: "github.com/<username>/blockchain-automation-framework.git"  
+            username: "git_username" 
+            password: "git_access_token" 
+            email: "git_email"                
+            private_key: "path_to_private_key" 
+
+          app:
+            google_maps_api: "api_key"
+          
+          services:
+            peers:
+            - peer:
+              name: carrier
+              subject: "O=Carrier,OU=Carrier,L=51.50/-0.13/London,C=GB" 
+              type: validator 
+              consortium_role: admin
+              geth_passphrase: 12345 
+              p2p:
+                port: 21000
+                ambassador: 15010 
+              rpc:
+                port: 8546
+                ambassador: 15011  
+              transaction_manager:
+                port: 8443  
+                ambassador: 8443 
+              raft: 
+                port: 50401
+                ambassador: 15013
+              db:  
+                port: 3306
+              expressapi:
+                port: 3000
+                targetPort: 3000
+              frontend:
+                targetPort: 80
+                port: 8080
+              geth_url: "http://carrier.test.corda.blockchaincloudpoc.com:15011" 
+      
+  pre_tasks:
+    - name: Create temp roles dir folder
+      file:
+        path: "{{ playbook_dir }}/../../shared/configuration"
+        state: directory
+        
+    - name: Copy the roles
+      copy:
+        src: "{{ playbook_dir }}/../../../../shared/configuration/roles"
+        dest: "{{ playbook_dir }}/../../shared/configuration"
+
+  tasks:
+    - name: "Copy openssl.conf to molecule directory"                 # Copying openssl.conf to molecule playbook directory
+      copy:
+        src: "{{ playbook_dir }}/../../openssl.conf"
+        dest: "{{ playbook_dir }}"
+        
+    - name: "Test ambassador-certs role"
+      include_role:
+        name: "configuration/roles/create/certificates/ambassador"
+      vars:
+        auth_lists:
```

Comment:
all of these variables are not needed. Please check the role `create/certificates/ambassador` and remove the vars that are not used